modules/apps/site/site-admin-web/src/main/resources/META-INF/resources/site_settings/analytics.jsp

```diff
@@ -26,19 +26,35 @@ for (String analyticsType : analyticsTypes) {
 
 	<c:choose>
 		<c:when test='<%= StringUtil.equalsIgnoreCase(analyticsType, "google") %>'>
-			<aui:input helpMessage="set-the-google-analytics-id-that-is-used-for-this-set-of-pages" label="google-analytics-id" name="googleAnalyticsId" type="text" value='<%= PropertiesParamUtil.getString(groupTypeSettings, request, "googleAnalyticsId") %>' />
+			<aui:field-wrapper cssClass="form-group">
+				<aui:input label="google-analytics-id" name="googleAnalyticsId" type="text" value='<%= PropertiesParamUtil.getString(groupTypeSettings, request, "googleAnalyticsId") %>' />
```

Comment:
This line adds a form input. Please check:

* The input is validated on the processing action

* The backend is protected agains value forging



Refer to documentation: